core/src/androidMain/kotlin/Filter.kt

```diff
@@ -0,0 +1,26 @@
+package com.juul.kable
+
+import com.benasher44.uuid.Uuid
+
+public actual sealed class Filter {
+
+    /**
+     * https://developer.android.com/reference/android/bluetooth/le/ScanFilter.Builder#setManufacturerData(int,%20byte[],%20byte[])
+     */
+    public class ManufacturerData(
+        /** A negative [id] is considered as invalid id. */
+        public val id: Int,
+
+        public val data: ByteArray,
+
+        /**
+         * For any bit in the mask, set it the 1 if it needs to match the one in [data], otherwise set it to 0.
```

Comment:
Nit:
```suggestion
         * For any bit in the mask, set it to 1 if it needs to match the corresponding bit in [data], otherwise set it to 0.
```